corehq/apps/ota/models.py

```diff
@@ -0,0 +1,66 @@
+from cStringIO import StringIO
+from django.db import models
+
+from casexml.apps.phone.restore import stream_response
+from corehq.blobs import get_blob_db
+from corehq.blobs.exceptions import NotFound
+
+import uuidfield
+
+
+class DemoUserRestore(models.Model):
+    uuid = uuidfield.UUIDField(unique=True, auto=True, db_index=True)
+    demo_user_id = models.CharField(max_length=255, default=None, db_index=True)
+    restore_blob_id = models.CharField(max_length=255, default=None, db_index=True)
+    timestamp_created = models.DateTimeField(auto_now=True)
```

Comment:
That will also create a new `DemoUserRestore` object. So that old version can be around if we want.